src/bitcoin.udl

```diff
@@ -82,6 +82,35 @@ dictionary TxOut {
   Script script_pubkey;
 };
 
+interface Transaction {
+  [Name=from_bytes, Throws=EncodeError]
+  constructor([ByRef] bytes transaction_bytes);
```

Comment:
The Rust equivalent here is `deserialize`. How do you feel about just keeping that? The `from_bytes` would be a sort of renaming of the method/constructor, and I'd like to keep those to a very minimum if possible, unless we make a very deliberate choice to start doing that.

Reply:
Sure. We might as well stick to their naming when possible as a convention